netket/hilbert/__init__.py

```diff
@@ -14,13 +14,16 @@
 
 from .abstract_hilbert import AbstractHilbert
 from .discrete_hilbert import DiscreteHilbert
+from .abstract_continuous_space_particle import AbstractParticle
```

Comment:
sorry to be pedantic, but lets call this file `continuous_space_abstract_particle`

Reply:
In view that the boson/fermion file is called continuous_space_boson/fermion doesn't it make more sense to have the abstract before continuous_space?

Reply:
if `DiscreteHilbert` was called `abstract_hilbert` yes.
maybe i have to rename that one.

Reply:
well ok then